zubhub_frontend/zubhub/src/views/home/Home.jsx

```diff
@@ -103,8 +102,8 @@ function Projects(props) {
           (
             <Box className={classes.root}>
               {hero && hero.id ? (
-                 <Box className={classes.heroSectionStyle}>
-                 <Box className={classes.heroContainerStyle}>
+                 <Box className={classes.heroMainSectionStyle}>
```

Comment:
Why have we renamed it to `heroMainSection` style instead of old name?

Reply:
There is another section on this page that uses the ```heroSectionStyle``` name for its classes. It is the carousel section for ambassadors and their testimonials. 
Making any changes to the already existing classes meant changing other sections as collateral too. However, since we were focusing on just the Hero Section and it required different styling than the rest of the components, I made a new class.